scripts/zones/Beadeaux/mobs/Ge_Dha_Evileye.lua

```diff
@@ -0,0 +1,49 @@
+-----------------------------------
+-- Area: Beadeaux (254)
+--  NM: Ge'Dha Evileye
+-- @spawnmob 17379450
+-- @pos -238 1 -202 (254)
```

Comment:
This sorta thing normally only gets used for force pop mobs, in the NPC's script not the mob. Just FYI.